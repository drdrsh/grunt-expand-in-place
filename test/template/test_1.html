<!DOCTYPE html>
<html>
<head>
	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
    <!-- expand_section ["assets/app/css/*.css", "assets/app/css/directory/*.css"] -->
	<!-- /expand_section -->
	<title>Simple title</title>
</head>
<body>
    <p>Some text</p>
	<!-- expand_section ["assets/app/tmpl/**/*.tmpl"] -->
	<!-- /expand_section -->

    <p>Some text</p>

	<!-- expand_section ["assets/app/js/**/*.js"] -->
	<!-- /expand_section -->

</body>
</html>
