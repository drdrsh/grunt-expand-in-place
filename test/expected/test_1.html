<!DOCTYPE html>
<html>
<head>
	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
    <!-- expand_section ["assets/app/css/*.css","assets/app/css/directory/*.css"] -->
	<link rel="stylesheet" type="text/css" href="./assets/app/css/frontend.css" />
	<link rel="stylesheet" type="text/css" href="./assets/app/css/directory/base.css" />
	<link rel="stylesheet" type="text/css" href="./assets/app/css/directory/date.css" />
	<link rel="stylesheet" type="text/css" href="./assets/app/css/directory/general.css" />
	<link rel="stylesheet" type="text/css" href="./assets/app/css/directory/hiddenfield.css" />
	<link rel="stylesheet" type="text/css" href="./assets/app/css/directory/list.css" />
	<link rel="stylesheet" type="text/css" href="./assets/app/css/directory/multi.css" />
	<link rel="stylesheet" type="text/css" href="./assets/app/css/directory/multil.css" />
	<link rel="stylesheet" type="text/css" href="./assets/app/css/directory/textarea.css" />
	<link rel="stylesheet" type="text/css" href="./assets/app/css/directory/textfield.css" />
	<!-- /expand_section -->
	<title>Simple title</title>
</head>
<body>
    <p>Some text</p>
	<!-- expand_section ["assets/app/tmpl/**/*.tmpl"] -->
	<script id="tmpl_base" type="text/html">This is a simple template</script>
	<script id="tmpl_date" type="text/html"></script>
	<script id="tmpl_hiddenfield" type="text/html"></script>
	<script id="tmpl_list" type="text/html"></script>
	<script id="tmpl_multi" type="text/html"></script>
	<script id="tmpl_multil" type="text/html"></script>
	<script id="tmpl_template" type="text/html"></script>
	<script id="tmpl_textarea" type="text/html"></script>
	<script id="tmpl_textfield" type="text/html"></script>
	<!-- /expand_section -->

    <p>Some text</p>

	<!-- expand_section ["assets/app/js/**/*.js"] -->
	<script type="text/javascript" src="./assets/app/js/form/core/form.js"></script>
	<script type="text/javascript" src="./assets/app/js/form/core/forms.js"></script>
	<script type="text/javascript" src="./assets/app/js/form/core/formset.js"></script>
	<script type="text/javascript" src="./assets/app/js/form/types/base.js"></script>
	<script type="text/javascript" src="./assets/app/js/form/types/date.js"></script>
	<script type="text/javascript" src="./assets/app/js/form/types/hiddenfield.js"></script>
	<script type="text/javascript" src="./assets/app/js/form/types/list.js"></script>
	<script type="text/javascript" src="./assets/app/js/form/types/multi.js"></script>
	<script type="text/javascript" src="./assets/app/js/form/types/multil.js"></script>
	<script type="text/javascript" src="./assets/app/js/form/types/textarea.js"></script>
	<script type="text/javascript" src="./assets/app/js/form/types/textfield.js"></script>
	<script type="text/javascript" src="./assets/app/js/template.js"></script>
	<!-- /expand_section -->

</body>
</html>
